crates/rbuilder/src/live_builder/block_output/relay_submit.rs

```diff
@@ -470,11 +499,11 @@ async fn submit_bid_to_the_relay(
             cancel.cancel();
         }
         Err(SubmitBlockErr::BidBelowFloor | SubmitBlockErr::PayloadAttributesNotKnown) => {
-            trace!(err = ?relay_result.unwrap_err(), "Block not accepted by the relay");
+            trace!(err = ?redactor.redact(relay_result.unwrap_err()), "Block not accepted by the relay");
```

Comment:
Maybe we can create our own wrapper around `trace!`, `error!`, etc that adds the redaction wrapper for us, it seems like an easy thing to forget to do.

Reply:
I thought about that but I don't know enough about macros and probably would take me a lot.
I would love to be able to put inside error! things like "if they try to log this type DON'T let it compile"